CONTRIBUTING.md

```diff
@@ -1,3 +1,32 @@
 # Contributing
 
-TODO
\ No newline at end of file
+First of all, we want to thank you for being willing to contribute to this project.
+
+
+## The most important part
+
+Before quality code, we mainly want a friendly environment. **PLEASE RESPECT** each person you might interact with, whether it's their first issue on GitHub or their thousandth pull request.
+
+Having a negative attitude leads to a stressful time, and we have so much in our lives we don't have time to care about being harassed online.
+
+
+## The technical part
+
+Here are the main directives to follow when contributing:
+
```

Comment:
I guess we should add this line
- if your PR is linked to an existing issue please refer to this one and close her after PR is merged